src/main/java/stringCalculator/Operator.java

```diff
@@ -0,0 +1,26 @@
+package stringCalculator;
+
+import java.util.function.BiFunction;
+
+public enum Operator {
+    PLUS("+", (first, second) -> first + second),
+    MINUS("-", (first, second) -> first - second),
+    MULTIPLY("*", (first, second) -> first * second),
+    DIVIDE("/", (first, second) -> first / second);
```

Comment:
```suggestion
    DIVIDE("/", (first, second) -> {
        if(second == 0){
            throw new IllegalArgumentException("0으로 나눌 수 없습니다.");
        }
        return first / second;
    });
```

나누기는 예외가 발생할 수 있는 케이스가 존재하는 것 같습니다.!